### PR TITLE
Add How To Use Navigation Layouts

### PR DESCRIPTION
**Pull request by ConnorLand**
Making this PR to collaborate with Chris Hodges. Will merge ASAP.

### CODE REVIEW

File: content/howto/front-end/use-navigation-layouts.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+5.  In the **General** tab, select **Shrinking content (initially open)** as the toggle mode:
+
+	{{% image_container width="350" %}}![](attachments/use-navigation-layouts/3.1_shrinkcontentopen.png){{% /image_container %}}
+
+### 3.2 Toggling the Sidebar for Shrinking Content (Initially Open)
+
+To toggle the sidebar for **Shrinking content (initially open)**, follow these steps:
+
+1.  Click **Run Locally** to run your project.
```
Comment (cjfhodges) [inline content/howto/front-end/use-navigation-layouts.md:122]: missing run the project screenshot 
attachments/use-navigation-layouts/3.2_runtheproject.png

File: content/howto/front-end/use-navigation-layouts.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+	{{% image_container width="350" %}}![](attachments/use-navigation-layouts/3.3_regionproperties.png){{% /image_container %}}
+
+5.  In the tab **General**, select **Shrinking content (initially closed)** as the toggle mode:
+
+	{{% image_container width="350" %}}![](attachments/use-navigation-layouts/3.3_shrinkcontentclosed.png){{% /image_container %}}
+
+### 3.4 Toggling the Sidebar for Shrinking Content (Initially Closed)
+
+1.  Click **Run Locally** to run your project.
```
Comment (cjfhodges) [inline content/howto/front-end/use-navigation-layouts.md:153]: missing run the project screenshot
attachments/use-navigation-layouts/3.4_runtheproject.png

**Review by cjfhodges — APPROVED** [review]
just need to clean up those __MACOSX/ files

File: content/howto/front-end/use-navigation-layouts.md (hunk 1 above)
Comment (ConnorLand) [reply content/howto/front-end/use-navigation-layouts.md:122]: Resolved over slack.

**Pull request merged**